com.stremio.Stremio.json

```diff
@@ -0,0 +1,204 @@
+{
+  "app-id": "com.stremio.Stremio",
+  "runtime": "org.kde.Platform",
+  "runtime-version": "5.12",
```

Comment:
```suggestion
  "runtime-version": "5.15",
```

Reply:
At least on my machine If I do this I get `Project ERROR: Unknown module(s) in QT: webengine` so this is no-no.

Reply:
Yes because the QtWebEngine thing is split into it's own baseapp and not included in the KDE SDK anymore. So this definitely can work.

Reply:
I've installed `io.qt.qtwebengine.BaseApp/x86_64/5.15` but no luck. How am I supposed to use it?

When I run `flatpak-builder --run _build com.stremio.Stremio.json sh -c 'updatedb 2> /dev/null;locate libqtwebengineplugin.so | grep -v /run/host'` there is no output. Maybe I need to do something more but I couldn't find any information online.

Reply:
You need to add something along this to your manifest:

```
base: io.qt.qtwebengine.BaseApp
base-version: "5.15"
```

(this is YAML version but you get the point)

Reply:
Changed the build system to CMake. Tested it and works with Qt 5.15.2